docs/getting_started/tutorial.md

```diff
@@ -0,0 +1,867 @@
+
+![](../_static/images/tutorial/tutorial_1.png)
+
+You can now access the administrative area at <http://127.0.0.1:8000/admin>
+
+![](../_static/images/tutorial/tutorial_2.png)
+
+## Extend the HomePage model
+
+Out of the box, the "home" app defines a blank `HomePage` model in `models.py`, along with a migration that creates a homepage and configures Wagtail to use it.
+
+Edit `home/models.py` as follows, to add a `body` field to the model:
+
+```python
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core.fields import RichTextField
+from wagtail.admin.edit_handlers import FieldPanel
+
+
+class HomePage(Page):
+    body = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('body', classname="full"),
+    ]
+```
+
+`body` is defined as `RichTextField`, a special Wagtail field. When `blank=True`,
+it means that this field is not required and can be empty. You
+can use any of the [Django core fields](https://docs.djangoproject.com/en/stable/ref/models/fields). `content_panels` define the
+capabilities and the layout of the editing interface. When you add fields to `content_panels`, it enables them to be edited on the Wagtail interface. [More on creating Page models](../topics/pages).
+
+Run `python manage.py makemigrations` (this will create the migrations file), then
+`python manage.py migrate` (this executes the migrations and updates the database with your model
+changes). You must run the above commands each time you make changes to
+the model definition.
+
+You can now edit the homepage within the Wagtail admin area (go to Pages, Homepage, then Edit) to see the new body field. Enter some text into the body field, and publish the page.
+
+The page template now needs to be updated to reflect the changes made
+to the model. Wagtail uses normal Django templates to render each page
+type. By default, it will look for a template filename formed from the app and model name,
+separating capital letters with underscores (e.g. HomePage within the 'home' app becomes
+`home/home_page.html`). This template file can exist in any location recognised by
+[Django's template rules](https://docs.djangoproject.com/en/stable/intro/tutorial03/#write-views-that-actually-do-something); conventionally it is placed under a `templates` folder within the app.
+
+Edit `home/templates/home/home_page.html` to contain the following:
+
+```html+django
+{% extends "base.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-homepage{% endblock %}
+
+{% block content %}
+    {{ page.body|richtext }}
+{% endblock %}
+
+``base.html`` refers to a parent template and must always be the first template tag used in a template. Extending from this template saves you from rewriting code and allows pages across your app to share a similar frame (by using block tags in the child template, you are able to override specific content within the parent template).
```

Comment:
Should these descriptions be below the `` ``` ``?

Reply:
Yup. Good catch. Thank you. 

Fixed in b0229fe52311d3cb240bd12233b6c540522c0a0e